app/views/layouts/application.html.erb

```diff
@@ -7,6 +7,12 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
+  <% if flash[:notice] %>
+    <br>
```

Comment:
Super nitpicky for front end, but generally the `<br>` tag isn't used just to create space, typically padding or margin would be added with CSS instead.